### ISSUE TITLE
Informative styles in Slider are not visible in high contrast mode

### ISSUE THREAD

**Issue by lloydstubber** [cultureamp/kaizen-design-system#1926]
TLDR; we need to make a border style for the Slider that is visible when in high contrast mode.

Type:Expert observation
Principle: Not applicable
Guideline:Not applicable
Success Criteria: Not applicable
Level:
Platform: Web
Severity:High

### What's the issue?
The rating sliders are not visible when viewed with Windows high contrast mode enabled, because it relies on a CSS property which is removed in this mode.

<img width="604" alt="Screen Shot 2021-09-02 at 4 37 46 pm" src="https://user-images.githubusercontent.com/18339250/131794645-46788397-347f-431c-baaf-1fbb65c876f5.png">

### Why it matters
People with low vision who rely on high contrast themes will miss out on this information.

### Our recommendation
Make sure that all informative styles are visible in Windows high contrast mode.
Use use a transparent border instead of relying on removed CSS properties such as background colours. This will only be visible when high contrast settings are enabled.
```
border: 1px solid transparent;
```
